yuuna/lib/src/utils/jidoujisho_localisations.dart

```diff
@@ -310,6 +312,321 @@ class JidoujishoLocalisations {
+    'Editando complementos para el perfil',
+    'creator_export_card': 'Crear Tarjeta',
+    'info_enhancements':
+    'Los complementos permite automatizar la edición de algún campo antes de la creación de la tarjeta.'
+        ' Seleccione una ranura a la derecha de un campo que permita el uso de un'
+        ' complemento. Se pueden utilizar hasta cinco ranuras para cada'
+        ' campo. El complemento en la ranura izquierda del campo será aplicado'
+        ' automáticamente en la creación de una tarjeta instantánea o al lanzamiento del Creador de'
+        ' Tarjetas.',
+    'info_actions':
+    'Quick actions allow for instant card creation and other automations'
+        ' to be used on dictionary search results. Actions can be assigned'
+        ' via the slots below. Up to six slots may be utilised.',
+    'no_more_available_enhancements':
+    'No hay complementos disponibles para este campo',
+    'no_more_available_quick_actions': 'No more available quick actions',
+    'assign_auto_enhancement': 'Assign Auto Enhancement',
+    'assign_manual_enhancement': 'Assign Manual Enhancement',
+    'remove_enhancement': 'Remover complemento',
+    'copy_of_mapping': 'Copia de %mapping_name%',
+    'enter_search_term': 'Ingrese un término a buscar...',
+    'searching_for': 'Buscando para 『%searchTerm%』...',
+    'no_search_results': 'No se encontraron resultados.',
+    'edit_actions': 'Editar Diccionario de Acciones Rápidas',
+    'remove_action': 'Remover Acción',
+    'assign_action': 'Asignar Acción',
+    'dictionary_import_tag': 'Imported from %dictionaryName%',
+    'stash_added_single': '『%term%』has been added to the Stash.',
+    'stash_added_multiple': 'Multiple items have been added to the Stash.',
+    'stash_clear_single': '『%term%』has been removed from the Stash.',
+    'stash_clear_title': 'Clear Stash',
+    'stash_clear_description': 'All contents will be cleared. Are you sure?',
+    'stash_placeholder': 'No items in the Stash',
+    'stash_nothing_to_pop': 'No items to be popped from the Stash.',
+    'no_sentences_found': 'No se encontraron oraciones',
+    'failed_online_service': 'Failed to communicate with online service',
+    'search_label_before': 'Resultado de busqueda ',
+    'search_label_middle': 'de ',
+    'search_label_after': 'encontrados para',
+    'clear_dictionary_title': 'Limpiar Historial de Resultados del Diccionario',
+    'clear_dictionary_description':
+    'This will clear all dictionary results from history. Are you sure?',
+    'clear_search_title': 'Limpiar Historial de Busqueda',
+    'clear_search_description':
+    'This will clear all search terms for this history. Are you sure?',
+    'clear_creator_title': 'Limpiar Creador',
+    'clear_creator_description': 'This will clear all fields. Are you sure?',
+    'copied_to_clipboard': 'Copiado al portapapeles.',
+    'no_text_to_segment': 'No text to segment.',
+    'info_fields': 'Fields are pre-filled based on the term selected on instant'
```

Comment:
I know long ones like these will be difficult, but they do stick out as some text is translated and others are not